dev-util/cmark/cmark-0.26.1.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="CommonMark parsing and rendering library and program in C"
+DESCRIPTION="cmark is the C reference implementation of CommonMark, \
+a rationalized version of Markdown syntax with a spec. \
+(For the JavaScript reference implementation, see commonmark.js.) It provides a shared library (libcmark) \
```

Comment:
The recipe has been updated to limit all the description lines to less than 80 characters.